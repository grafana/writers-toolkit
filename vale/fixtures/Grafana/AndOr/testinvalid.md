Don't use:

- You can view and/or edit your own data.
- You can export raw and/or processed events.
