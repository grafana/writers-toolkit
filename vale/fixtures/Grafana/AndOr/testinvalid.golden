testinvalid.md:3:16:Grafana.AndOr:Avoid writing and/or except when space is limited, such as in tables. Often, 'and' implies 'or', so you don't need to write both words. If you need to specify both in your content, write something like "You can export raw events, processed events, or both." 
testinvalid.md:4:22:Grafana.AndOr:Avoid writing and/or except when space is limited, such as in tables. Often, 'and' implies 'or', so you don't need to write both words. If you need to specify both in your content, write something like "You can export raw events, processed events, or both." 
